pallets/rmrk-core/src/lib.rs

```diff
@@ -130,7 +163,7 @@ pub mod pallet {
 	#[pallet::getter(fn nfts)]
 	/// Stores nft info
 	pub type Nfts<T: Config> =
-		StorageDoubleMap<_, Twox64Concat, CollectionId, Twox64Concat, NftId, InstanceInfoOf<T>>;
+		StorageDoubleMap<_, Twox64Concat, CollectionId, Twox64Concat, T::ItemId, InstanceInfoOf<T>>;
```

Comment:
The changes for `NftId` to `T::ItemId` was mentioned on call, but what was the reason it was causing the build failures or does this make code easier to maintain? I notice it simplifies `impl` from needing to add the `where` clause that was redundant before.

Reply:
The compiler would report that `benchmark!` macro is not expecting last `}`
Which didn't make sense. Removing this `where` clause solved the problem.
Also please read the this PR's description.